velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -142,11 +142,14 @@ const ::substrait::Expression& VeloxToSubstraitExprConvertor::toSubstraitExpr(
         substraitExpr->mutable_scalar_function();
 
     // TODO need to change yaml file to register function, now is dummy.
-
-    scalarExpr->set_function_reference(functionMap_[functionName]);
-
+    if (functionMap_.find(functionName) == functionMap_.end()) {
```

Comment:
This change seems unrelated. It would be nice to move it into a separate PR. 

Also, use VELOX_CHECK and fix the error message:

`VELOX_CHECK(functionMap_.find(functionName) != functionMap_.end(), "Scalar function not found: {}", functionName)`